src/SFA.DAS.EmployerFinance.Web/Views/Shared/_GoogleAnalyticsBody.cshtml

```diff
@@ -0,0 +1,12 @@
+@using SFA.DAS.EmployerFinance.Configuration
+@{
```

Comment:
Is the view bag property needed? You should be able to use `@inject GoogleAnalyticsConfiguration configuration`.

Reply:
changed